packages/zosuss/src/Shell.ts

```diff
@@ -94,8 +94,14 @@ export class Shell {
                                 dataToPrint = "";
                                 isUserCommand = false;
                             } else if (isUserCommand) {
-                                // print out the user command result
-                                stdoutHandler(dataToPrint);
+                                // don't print out command (starts with `$`) just print result
+                                if (!dataToPrint.match(/\$/)){
+                                    if (dataToPrint.startsWith("\r\n")){
+                                        stdoutHandler(dataToPrint.split("\r\n")[1]);
```

Comment:
I'm afraid this might be removing any new line endings coming back, which messes up with the output 😢 

<details>
<summary>Before and After with the current implementation</summary>
Before:

```
zFernand0 @ SSH:~/gh# zowe uss issue cmd "ls -al clitest/test" 

$ total 48
drwxr-xr-x   3 OMVSKERN SYS1        8192 Jul 26 15:07 .
drwxr-xr-x   8 OMVSKERN SYS1        8192 Jul 27 11:54 ..
drwxr-xr-x   4 OMVSKERN SYS1        8192 Jul 26 17:49 usstest
zFernand0 @ SSH:~/gh# zowe uss issue cmd "ls -al clitest/test" --rfj
{
  "success": true,
  "exitCode": 0,
  "message": "",
  "stdout": "\r\n$ total 48\r\ndrwxr-xr-x   3 OMVSKERN SYS1        8192 Jul 26 15:07 .\r\ndrwxr-xr-x   8 OMVSKERN SYS1        8192 Jul 27 11:54 ..\r\ndrwxr-xr-x   4 OMVSKERN SYS1        8192 Jul 26 17:49 usstest\n",
  "stderr": "",
  "data": {}
}
```

After

```
zFernand0 @ SSH:~/gh# zdev uss issue cmd "ls -al clitest/test" 
drwxr-xr-x   3 OMVSKERN SYS1        8192 Jul 26 15:07 .
zFernand0 @ SSH:~/gh# zdev uss issue cmd "ls -al clitest/test" --rfj
{
  "success": true,
  "exitCode": 0,
  "message": "",
  "stdout": "drwxr-xr-x   3 OMVSKERN SYS1        8192 Jul 26 15:07 .\n",
  "stderr": "",
  "data": {}
}
```
</details>

I believe what we wanted to do is to remove the first line that starts with `\r\n`?
perhaps `slice()` (and re-`join`ing the array) would help 😋 

<details>
<summary>Using the `slice` and `join` trick</summary>
Before:

```
zFernand0 @ SSH:~/gh# zowe uss issue cmd "ls -al clitest/test" 

$ total 48
drwxr-xr-x   3 OMVSKERN SYS1        8192 Jul 26 15:07 .
drwxr-xr-x   8 OMVSKERN SYS1        8192 Jul 27 11:54 ..
drwxr-xr-x   4 OMVSKERN SYS1        8192 Jul 26 17:49 usstest
zFernand0 @ SSH:~/gh# zowe uss issue cmd "ls -al clitest/test" --rfj
{
  "success": true,
  "exitCode": 0,
  "message": "",
  "stdout": "\r\n$ total 48\r\ndrwxr-xr-x   3 OMVSKERN SYS1        8192 Jul 26 15:07 .\r\ndrwxr-xr-x   8 OMVSKERN SYS1        8192 Jul 27 11:54 ..\r\ndrwxr-xr-x   4 OMVSKERN SYS1        8192 Jul 26 17:49 usstest\n",
  "stderr": "",
  "data": {}
}
```

After

```
zFernand0 @ SSH:~/gh# zdev uss issue cmd "ls -al clitest/test" 

drwxr-xr-x   3 OMVSKERN SYS1        8192 Jul 26 15:07 .
drwxr-xr-x   8 OMVSKERN SYS1        8192 Jul 27 11:54 ..
drwxr-xr-x   4 OMVSKERN SYS1        8192 Jul 26 17:49 usstest
zFernand0 @ SSH:~/gh# zdev uss issue cmd "ls -al clitest/test" --rfj
{
  "success": true,
  "exitCode": 0,
  "message": "",
  "stdout": "\r\ndrwxr-xr-x   3 OMVSKERN SYS1        8192 Jul 26 15:07 .\r\ndrwxr-xr-x   8 OMVSKERN SYS1        8192 Jul 27 11:54 ..\r\ndrwxr-xr-x   4 OMVSKERN SYS1        8192 Jul 26 17:49 usstest\n",
  "stderr": "",
  "data": {}
}
```
</details>


The only problem with that is that we can't assume that removing the first line is the correct solution either. Let's try the command `pwd`.

<details>
<summary>Using the `pwd` command as an example</summary>
Before:

```
zFernand0 @ SSH:~/gh# zowe uss issue cmd "pwd" 

$ /u/users/rijfe01
zFernand0 @ SSH:~/gh# zowe uss issue cmd "pwd" --rfj
{
  "success": true,
  "exitCode": 0,
  "message": "",
  "stdout": "\r\n$ /u/users/rijfe01\n",
  "stderr": "",
  "data": {}
}
```

After

```
zFernand0 @ SSH:~/gh# zdev uss issue cmd "pwd" 

zFernand0 @ SSH:~/gh# zdev uss issue cmd "pwd" --rfj
{
  "success": true,
  "exitCode": 0,
  "message": "",
  "stdout": "\n",
  "stderr": "",
  "data": {}
}
```
</details>

So, we may need to find a better way to handle these special cases 😋